practice/lib/regex/functionality/regex_patterns.rb

```diff
@@ -0,0 +1,10 @@
+module RegexPatterns
+  # Path to the file directory, file's name is given as argument in methods below
+  FILE_PATH = File.join(Dir.home, 'RubyTasks', 'practice', 'lib', 'regex', 'test_data').freeze
+
+  # Method should return amount of occurrences of the sting
+  def self.count_occurrence(file_name, regex_pattern)
+    document = File.join(FILE_PATH, file_name)
```

Comment:
What a problem I see here that we won't be able to reuse this method for files which are not in test_data folder.
This method should get full path to file as an argument but not just a file name